frontend/assets/javascripts/src/modules/metrics/triggers.js

```diff
@@ -13,7 +13,7 @@ define([
 
     function init() {
         var trackingElems = document.querySelectorAll('[' + METRIC_ATTRS.trigger + ']');
-        if (window.ga && trackingElems) {
+        if (trackingElems) {
```

Comment:
The logger function also has its own window.ga presence check before actually calling the send function on window.ga, so the event handler bindings can still happen regardless of whether Google is loaded or not.  Hence a 👍 from me.